docs/DeveloperGuide.adoc

```diff
@@ -483,6 +483,8 @@ Auto-suggestion of command arguments while the user keys in inputs can be very h
 Possible command words will be suggested to user based on incomplete input.
 Depending on what the user has typed in for the argument, the most similar argument values retrieved from the historical
 records will be displayed to the user for reference.
+The user can also use tab to auto complete the command word/argument, where the first prompt will
```

Comment:
Make it clear you are referring to the "Tab" key to avoid confusion with the tabs in the UI.